src/test/ComponentTestPage.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <Accordion>
+    <AccordionItem v-for="item of accordionTestItems" :key="item.title">
+      <template v-slot:title>
```

Comment:
Accordion still operates off slots, 2 slots, 1 for the title area, and 1 for the body content.